### ISSUE TITLE
Move orphaned role tests into tentative files

### ISSUE THREAD

**Issue by nmlapre** [web-platform-tests/interop#645]
### Test List

- in [grid-roles.html](https://wpt.fyi/results/wai-aria/role/grid-roles.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
  - orphaned button with gridcell role outside the context of row
  - orphaned row outside the context of table
  - orphaned rowgroup outside the context of row
  - orphaned div with gridcell role outside the context of row
  - orphaned rowheader outside the context of row
  - orphaned columnheader outside the context of row
- in [list-roles.html](https://wpt.fyi/results/wai-aria/role/list-roles.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
  - orphan p with listitem role
- in [listbox-roles.html](https://wpt.fyi/results/wai-aria/role/listbox-roles.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
  - orphaned option outside the context of listbox
- in [menu-roles.html](https://wpt.fyi/results/wai-aria/role/menu-roles.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
  - orphaned menuitem outside the context of menu/menubar
  - orphaned menuitemradio outside the context of menu/menubar
  - orphaned menuitemcheckbox outside the context of menu/menubar
  - orphan button with menuitem role
  - orphan button with menuitemcheckbox role
  - orphan button with menuitemradio role
- in [tab-roles.html](https://wpt.fyi/results/wai-aria/role/tab-roles.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
  - orphan button with tab role
- in [tree-roles.html](https://wpt.fyi/results/wai-aria/role/tree-roles.html?label=master&label=experimental&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
  - orphaned treeitem outside the context of tree
  - orphaned button with treeitem role outside tree context

### Rationale

Please see rationale in https://github.com/web-platform-tests/interop-accessibility/issues/109.

Since these tests aren't currently supported by the spec, I think we should move them to a `.tentative` file or files.

**Comment by foolip** [comment]
Is the proposal just to rename the tests to make clear they're not backed by a spec, or to also drop them from Interop 2024? We have on a few occasions included tentative tests, but it's worth flagging when we do that since the tests themselves are then the only guide to implementation.

**Comment by nairnandu** [comment]
@aleventhal could you please review the proposal for Chromium?
@nt1m could you please review for WebKit?

**Comment by nt1m** [comment]
@cookiecrook Can you help review for WebKit?

**Comment by aleventhal** [comment]
I agree that there is no requirement for browsers to ignore orphaned roles where they are missing a required ancestor, etc.
If we're ever going to make that a requirement it needs to be discussed in ARIA WG because of potential performance and behavioral regressions.

**Comment by nmlapre** [comment]
> Is the proposal just to rename the tests to make clear they're not backed by a spec, or to also drop them from Interop 2024?

I'd prefer to drop the tests from Interop 2024, unless [the ARIA spec issue](https://github.com/w3c/aria/issues/2137) is resolved quite soon. I was under the impression that renaming with `.tentative` was tantamount to removal for Interop 2024 (see comment [here](https://github.com/web-platform-tests/interop-accessibility/issues/109#issuecomment-1979883767)), but maybe I'm wrong on that. I'm happy to pull them out entirely.

**Comment by cookiecrook** [comment]
As mentioned in https://github.com/web-platform-tests/wpt/pull/45032, there are spec requirements in Core-AAM for this. The discussion is happening there, so I don’t think these tests will be pulled out of the Focus Area, until implementors wish to change Core-AAM. 

There’s also an open ARIA Issue and an Interop-Accessibility issue:
- https://github.com/w3c/aria/issues/2137
- https://github.com/web-platform-tests/interop-accessibility/issues/109
so this one can be closed I think.
